src/main/java/weavers/siltarae/global/exception/ExceptionCode.java

```diff
@@ -7,6 +7,7 @@
 @RequiredArgsConstructor
 public enum ExceptionCode {
     INVALID_REQUEST(1000, "올바르지 않은 요청입니다."),
+    DUPLICATED_TAG_NAME(1001, "중복된 태그명입니다."),
```

Comment:
Tag 관련해서 1000번대의 에러코드를 사용하신 이유가 있을까요?

Reply:
제 생각에는 태그오류와 올바르지않는 요청 오류를 구분해서 태그는 3000번대로 가는 방향도 좋을 것 같아요!

Reply:
예외코드는 아직 팀원 간에 합의된 사항이 없는 상태라서 1000번대를 지정했는데 도메인별로 구분하는게 더 좋을 것 같네요!
[이 포스팅](https://cheese10yun.github.io/spring-guide-exception/) 참고하여 커스텀 에러 코드 설계하는 건 어떨까요?

Reply:
위 링크 참고해서 지정하면 이렇게 될 것 같네요!

```suggestion
    DUPLICATED_TAG_NAME(3001, "중복된 태그명입니다."),
```